release.sh

```diff
@@ -35,11 +35,12 @@ SKIP_CHECKSTYLE="-Dcheckstyle.ant.skip=true -Dcheckstyle.skip=true"
 SKIP_OTHERS="-Dpmd.skip=true -Dspotbugs.skip=true -Djacoco.skip=true -Dxml.skip=true"
 
 echo "Version bump in pom.xml (release:prepare) ..."
-mvn -e --no-transfer-progress -Pgpg release:prepare -B -Darguments="$SKIP_TEST $SKIP_CHECKSTYLE \
-  $SKIP_OTHERS"
+mvn -e --no-transfer-progress -Pgpg release:prepare -B \
+  -Darguments="$SKIP_TEST $SKIP_CHECKSTYLE $SKIP_OTHERS"
```

Comment:
Why not just use the `no-validations` profile? You will basically end up reproducing it if you want to exclude everything.

Reply:
there are nuances.
we should not skip javadoc for example. I remember something else was not ok to use it. 

We might need to make special profile for release, but it is not very easy to experiment on release process - recovering from release failure is mess. I think in december I will be have time to focus heavily on release to optimize it more.

Reply:
Even with profile, you can turn things on that profile turns off. It can also work with double profiles.
Example:
https://github.com/sevntu-checkstyle/sevntu.checkstyle/blob/master/.ci/validation.sh#L50

Reply:
If you need to test a release to maven folder, you can try:
https://github.com/rnveach/checkstyle-backport-jre8/blob/master/pom.xml#L656

Reply:
we will try at https://github.com/checkstyle/checkstyle/issues/12079